tntconnect.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+export WINEPREFIX="/var/data/tntconnect"
+export WINEARCH="win32"
+export WINEDLLOVERRIDES="msado15=n,b"
+export XDG_CACHE_HOME="/app/tmp"
```

Comment:
a cache on a read only location?

Reply:
Yes :smile:  This causes `winetricks` to use the already downloaded files from the cache instead of downloading again.

The first step when the user starts the app for the first time is the installation of some missing components. Unfortunately this can't be done at build time because it brings up a dialog (even with unattended install) and I found no way around that. But this way at least the user doesn't have to wait for the components to download...